src/documents/activities/ToddlerGrandparentConversation.html.md.handlebars

```diff
@@ -0,0 +1,25 @@
+---
+title: The Toddler-Grandparent Conversation
+layout: content
+category: Activities
+---
+
+Toddlers repeatedly ask “why?”. Why is the sky blue? Why do cats have tails? In this activity, we participate in an imagined grandparent-toddler conversation by repeatedly asking “why?”. 
+
+This activity is a great way to unpack the motivations or the nested context of a design scenario. It demonstrates and exercises the third dimension of inclusive design – Broader Beneficial Impact – by taking us from the local and immediate assumptions out to their larger framing and context. 
```

Comment:
I think the word "out" is what makes this sentence awkward for me. What if we keep the original structure and remove the "out"? I think it will still work.

Otherwise I think we should use "in from / out to" sentence structure.